### ISSUE TITLE
About loss_type=None

### ISSUE THREAD

**Issue by IamRecalcitrance** [modelscope/ms-swift#4937]
`loss_type=None` was set in the config but it is unrecognised.Using the default loss: `ForCausalLMLoss`.

我发现这个警告怎么也消不掉，并不是我想使用的损失函数，这是启动脚本：

nproc_per_node=8

# losses: plugin/loss.py
# 8*40G
MAX_PIXELS=1003520 \
NPROC_PER_NODE=$nproc_per_node \
swift sft \
    --model gme-Qwen2-VL-7B-Instruct \
    --train_type lora \
    --dataset 'ms-swift-main/dataset' \
    --torch_dtype bfloat16 \
    --num_train_epochs 5 \
    --per_device_train_batch_size 4 \
    --per_device_eval_batch_size 1 \
    --gradient_accumulation_steps $(expr 64 / $nproc_per_node) \
    --eval_steps 100 \
    --save_steps 100 \
    --eval_strategy steps \
    --save_total_limit 5 \
    --logging_steps 1 \
    --output_dir output \
    --lazy_tokenize true \
    --warmup_ratio 0.05 \
    --learning_rate 5e-6 \
    --deepspeed zero3 \
    --dataloader_num_workers 4 \
    --task_type embedding \
    --loss_type online_contrastive \
    --split_dataset_ratio 0.2 \
    --dataloader_drop_last true

**Comment by Jintao-Huang** [comment]
这个警告忽略就好了

**Comment by IamRecalcitrance** [comment]
> 这个警告忽略就好了

好的。不会产生实际影响吗？我还以为它真给我换了个损失函数

**Comment by viviansmlie** [comment]
> > 这个警告忽略就好了
> 
> 好的。不会产生实际影响吗？我还以为它真给我换了个损失函数

我也以为把loss换了呢😂
